### PR TITLE
straight to virtuoso for the publishing (experiment)

### PR DESCRIPTION
**Pull request by nbittich**
I keep it in draft PR because it is still experimental , to test add VIRTUOSO_SPARQL_ENDPOINT: "http://virtuoso:8890/sparql" as environment variable

### CODE REVIEW

**Comment by cecemel** [comment]
I am not sure i understand the PR. It seems a bigger change then *just* having a run time option to say: on publish, change target host?

File: lib/sparql-virtuoso.js
Comment (cecemel) [inline lib/sparql-virtuoso.js:1]: I don't see what's it doing here. Starting from 0.6 mu-auth-sudo has all the options it need to go to other hosts

Comment (nbittich) [reply lib/sparql-virtuoso.js:1]: Ah, didn't know that

**Comment by nbittich** [comment]
> I am not sure i understand the PR. It seems a bigger change then _just_ having a run time option to say: on publish, change target host?

goal was to test how fast virtuoso is compared to mu auth, this shouldn't be merged; 

It seems to be  10-100 times faster with direct queries to virtuoso. I think for some urls (e.g  ` https://ebesluitvorming.gent.be/zittingen/lijst;jsessionid=DCDB7D548BC02F63FB62AD695C5B7681?month=06&year=2023` )  the only solution might be to use virtuoso directly, otherwise it would take way too long (probably week(s) )

I'm still not sure what would be the consequences (if that's ok to bypass mu auth and skip delta production)

**Pull request closed without merging**